test/mako/alerter/github/issue.go

```diff
@@ -87,65 +88,72 @@ func Setup(org, repo, githubTokenPath string, dryrun bool) (*IssueHandler, error
 // CreateIssueForTest will try to add an issue with the given testName and description.
 // If there is already an issue related to the test, it will try to update that issue.
 func (gih *IssueHandler) CreateIssueForTest(testName, desc string) error {
-	org := gih.config.org
-	repo := gih.config.repo
-	dryrun := gih.config.dryrun
 	title := fmt.Sprintf(issueTitleTemplate, testName)
-	issue := gih.findIssue(org, repo, title, dryrun)
+	issue, err := gih.findIssue(title)
+	if err != nil {
+		return fmt.Errorf("failed to find issues for test %q, skipped creating new issue", testName)
+	}
 	// If the issue hasn't been created, create one
 	if issue == nil {
-		body := fmt.Sprintf(issueBodyTemplate, testName)
-		issue, err := gih.createNewIssue(org, repo, title, body, dryrun)
+		commentBody := fmt.Sprintf(issueBodyTemplate, testName, gih.config.repo)
+		issue, err := gih.createNewIssue(title, commentBody)
 		if err != nil {
-			return err
-		}
-		comment := fmt.Sprintf(newIssueCommentTemplate, desc)
-		if err := gih.addComment(org, repo, *issue.Number, comment, dryrun); err != nil {
-			return err
-		}
-		// If one issue with the same title has been closed, reopen it and add new comment
-	} else if *issue.State == string(ghutil.IssueCloseState) {
-		if err := gih.reopenIssue(org, repo, *issue.Number, dryrun); err != nil {
-			return err
+			return fmt.Errorf("failed to create a new issue for test %q", testName)
 		}
-		comment := fmt.Sprintf(reopenIssueCommentTemplate, desc)
-		if err := gih.addComment(org, repo, *issue.Number, comment, dryrun); err != nil {
-			return err
+		commentBody = fmt.Sprintf(issueSummaryCommentTemplate, desc)
+		if err := gih.addComment(*issue.Number, commentBody); err != nil {
+			return fmt.Errorf("failed to add comment for new issue %d", *issue.Number)
 		}
 	} else {
```

Comment:
perhaps, `return nil` and unindent here?

Reply:
Done.